src/pubkey.h

```diff
@@ -205,4 +210,13 @@ struct CExtPubKey {
     bool Derive(CExtPubKey& out, unsigned int nChild) const;
 };
 
+/** Users of this module must hold an ECCVerifyHandle. The constructor and
+ *  destructor of these are not allowed to run in parallel, though. */
```

Comment:
Yes, if we pass it through main, consensus, script interpreter, and
pubkey... which is possible but annoyingly much code for something that's
just an immutable blob initialized at startup and usable from all threads.